src/testsuite/single/tests/efuns/replaceable_1.c

```diff
@@ -0,0 +1,8 @@
+inherit "/testsuite/single/tests/efuns/replaceable_inh";
```

Comment:
since everything placed under /test/efuns/ will get build and run, maybe you want to move those into /clone/ instead?

but not important.